packages/blade/src/_helpers/storybook/ComponentStatusTable.tsx

```diff
@@ -0,0 +1,411 @@
+  },
+  {
+    name: 'Tooltip',
+    status: 'planned-Q4-design',
+    description:
+      'Tooltip is a brief, informative message that appears when a user interacts with an element.',
+  },
+];
+
+const Table = styled.table`
+  width: 100%;
+  border-collapse: collapse;
+  table-layout: fixed;
+
+  th,
+  td {
+    width: 20%;
+    padding: ${({ theme }) => makeSpace(theme.spacing[4])};
+    padding-inline: ${({ theme }) => makeSpace(theme.spacing[7])};
+  }
+
+  tr:nth-child(odd) {
+    background-color: ${({ theme }) => theme.colors.brand.gray[200].lowContrast};
+  }
+
+  tr:nth-child(even) {
+    background-color: ${({ theme }) => theme.colors.brand.gray[300].lowContrast};
+  }
+`;
+
+const ComponentStatusBadge = ({ status }: { status: ComponentStatuses }): React.ReactElement => {
+  const badgeVariants: Record<
+    ComponentStatuses,
+    { label: string; variant: BadgeProps['variant']; icon: IconComponent }
+  > = {
+    released: { label: 'Released', variant: 'positive', icon: CheckIcon },
+    deprecated: { label: 'Deprecated', variant: 'negative', icon: AlertCircleIcon },
+    'in-design': { label: 'In Design', variant: 'notice', icon: LoaderIcon },
+    'in-api-spec': { label: 'API In Progress', variant: 'notice', icon: EditIcon },
+    'in-development': { label: 'In Development', variant: 'notice', icon: LoaderIcon },
+    'planned-Q1-dev': { label: 'Planned: Q1 Dev', variant: 'information', icon: ClockIcon },
+    'planned-Q2-dev': { label: 'Planned: Q2 Dev', variant: 'information', icon: ClockIcon },
+    'planned-Q3-dev': { label: 'Planned: Q3 Dev', variant: 'information', icon: ClockIcon },
+    'planned-Q4-dev': { label: 'Planned: Q4 Dev', variant: 'information', icon: ClockIcon },
+    'planned-Q1-design': { label: 'Planned: Q1 Design', variant: 'information', icon: ClockIcon },
+    'planned-Q2-design': { label: 'Planned: Q2 Design', variant: 'information', icon: ClockIcon },
+    'planned-Q3-design': { label: 'Planned: Q3 Design', variant: 'information', icon: ClockIcon },
+    'planned-Q4-design': { label: 'Planned: Q4 Design', variant: 'information', icon: ClockIcon },
+  };
+
+  return (
+    <Badge variant={badgeVariants[status].variant} icon={badgeVariants[status].icon}>
+      {badgeVariants[status].label}
+    </Badge>
+  );
+};
+
+const ReleasedInLink = ({ version }: { version?: string }): React.ReactElement => {
+  const ghUrl = 'https://github.com/razorpay/blade/releases/tag/%40razorpay%2Fblade%40';
+  return version ? (
+    <Link
+      href={`${ghUrl}${version}`}
+      rel="noopener noreferrer"
+      target="_blank"
+    >{`v${version}`}</Link>
+  ) : (
+    <Text>-</Text>
+  );
+};
+
+const ComponentStatusTable = (): React.ReactElement => {
+  const sortedData = componentData.sort((a, b) => {
+    if (!a.releasedIn || !b.releasedIn) {
+      return a.status.localeCompare(b.status);
+    }
+    return a.name.localeCompare(b.name);
```

Comment:
Finding it difficult to sort without groups like so, because js `sort(a, b)` can only sort upto 3 groups 0, 1, -1

- Unreleased components
  - in-dev, in-api-spec
  - planned-dev
  - planned-design
- Released components

Reply:
So left it as is: 

<img width="202" alt="image" src="https://user-images.githubusercontent.com/35374649/215071377-f78328e9-dc30-4bd2-bfe6-ec87d7a9b648.png">